src/index.ts

```diff
@@ -552,12 +552,20 @@ class Logging {
     opts?: GetEntriesRequest | GetEntriesCallback
   ): Promise<GetEntriesResponse> {
     const options = opts ? (opts as GetEntriesRequest) : {};
-    const reqOpts = extend(
-      {
-        orderBy: 'timestamp desc',
-      },
-      options
-    );
+
+    // By default, sort entries by descending timestamp
+    let reqOpts = extend({orderBy: 'timestamp desc'}, options);
+
+    // By default, filter entries to last 24 hours only
```

Comment:
What was the previous behavior?

Reply:
Previously, it would get as many entries as possible until a max threshold was reached. It took about 10 minutes. The default behavior across CLI is actually to filter by 24 hours, as well as what's recommended in documentation. 
We were getting a lot of support tickets across different language repos on this.
This fix reduces query time to a few seconds.

Reply:
Sounds good, I just wanted to get that information to determine if it's a breaking change, which it sounds like it is. What do you think?

Reply:
Great question, this actually got a lot of debate. After some back and forth in the chats, we decided to mark it as breaking. and bundle it with another breaking fix that just got merged last night

Reply:
Great, thanks!